File: backend/local-api-module/src/main/java/org/vyatsu/localApiModule/dto/response/api/RoleDto.java
package org.vyatsu.localApiModule.dto.response.api;

import lombok.Value;
import org.vyatsu.localApiModule.entity.enums.RoleType;

import java.io.Serializable;

/**
 * DTO for {@link org.vyatsu.localApiModule.entity.role.Role}
 */
@Value
public class RoleDto implements Serializable {
    Long id;
    RoleType name;
}

File: app/src/main/java/com/bytebusters/android/mpsystem/dto/response/api/RoleDto.kt
package com.bytebusters.android.mpsystem.dto.response.api

import com.bytebusters.android.mpsystem.entity.enums.RoleType
import java.io.Serializable

class RoleDto (
    var id: Long? = null,
    var name: RoleType? = null
)